services/frontend-service/src/ui/components/SideBar/SideBar.scss

```diff
@@ -57,6 +57,35 @@
     @extend .text-regular;
 }
 
+.mdc-drawer-sidebar-list-item-delete-all {
+    @extend .text-regular;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    border-top: solid;
+    padding-top: 10px;
+    margin-top: 10px;
```

Comment:
Why do we use both margin and padding? (if there is no difference, I'd rather use padding only (or margins only) and not both at the same time)

Reply:
the difference is that we display the border here, and want the same spacing on the inside and outside.

Reply:
Okay, I have missed that part. Thank you for explaining :+1: